File: Assignment5/program1.c
#include<stdio.h>
// TIME COMPLEXITY = O(N)  
void Pattern(int iNo)
{
  int iCnt=0;
  if(iNo < 0)
  {
    iNo = -iNo;
  }
  for(iCnt=1;iCnt<=iNo;iCnt++)
  {
    printf("$ * ");
  }
  
}

int main()
{
  int iValue = 0;
  printf("Enter the number : ");
  scanf("%d",&iValue);
  Pattern(iValue);
  return 0;
}

File: Assignment11/program1.java
import java.util.Scanner;
// TIME COMPLEXITY O(N)
class Pattern
{
  public void Display(int iCol)
  {
    int i = 0;
    char ch ='\0';
    
    for(i = 1 ,ch='A'; i<=iCol;i++,ch++)
    {
      System.out.print(ch+"\t");
      
    }
  }
}
public class program1 
{
  public static void main(String[] args)
   {
      int iValue = 0;
      Scanner sobj = new Scanner(System.in);
      System.out.println("Enter the number of Elements: ");
      iValue = sobj.nextInt();
      Pattern pobj = new Pattern();
      pobj.Display(iValue);

  }
}
